feature-utils/poly-analysis/src/context/poly-analysis.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext, useEffect, useState } from "react";
+
+export const PolyAnalysisContext = createContext();
+
+export const PolyAnalysisProvider = ({
```

Comment:
We should probably commit to document all these new components pretty thoroughly. Looks fine initially, but we're now a big team and it's not easy to keep up to date with all new developments.